### DISCUSSION TITLE
Trim whitespaces during license validation

### DISCUSSION THREAD

**Discussion by haztakki** [spatie/ray#66] [Ideas]
When entering a license in the app, you have to be careful of whitespaces or else you are told the license is invalid. It would be useful if whitespaces were trimmed away upon validating. Additionally it would be a nice UI/UX touch to add a copy to clipboard button under the product licenses page in the Spatie dashboard.

**Comment by freekmurze** [comment]
Thanks, we'll get this sorted soon!

**Comment by AdrianMrn** [comment]
I added both the trim and a copy button for the licenses, thanks for the suggestion!

**Reply by haztakki** [reply]
Great work. Looking forward to checking it out in the next version. 😀
